docs/TektonConfig.md

```diff
@@ -650,3 +655,12 @@ The following fields are supported in `HorizontalPodAutoscaler` (aka HPA)
 [schedule]:https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax
 [priorityClassName]: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority
 [priorityClass]: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass
+
+### webhookConfigurationOptions
+Defines additional options for pipelines webooks, To get detailed information about webhooks options visit https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/
```

Comment:
I think this is valid for all webhook, not only pipelines, basically in any component.

Also, we should mention that it will be applied only if some webhook exists with this name, otherwise ignored.

Reply:
done